.github/ISSUE_TEMPLATE/release.md

```diff
@@ -38,9 +41,11 @@ $ go build -tags tti ./cmd/tti-lw-cli
 - [ ] Then export the CLI documentation using the following command:
 
 ```bash
-$ HOME=$HOME ./tti-lw-cli gen-yaml-doc -o /path/to/lorawan-stack-docs/doc/data/
+$ HOME='$HOME' ./tti-lw-cli gen-yaml-doc -o ../lorawan-stack-docs/doc/data/
```

Comment:
What is this `HOME` assignment trying to do?

Reply:
Making sure that the documentation of the `--config` flag stays `.ttn-lw-cli.yml,$HOME/.ttn-lw-cli.yml,$HOME/Library/Application Support/.ttn-lw-cli.yml` and doesn't expand to the home of whoever generates the documentation.